pages/tutorials/3.0/system/angle.md

```diff
@@ -0,0 +1,81 @@
+# Handling Angles
+
+## Angles in SFML
+
+SFML defines angles as following the [right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule).
+This means that if the X-axis points to the right in the window and the Y-axis points down, then a positive angle rotates clockwise.
+Encoding these angles in code is handled by [`sf::Angle`](https://www.sfml-dev.org/todo/angle.php "sf::Angle documentation").
+All SFML classes that use angles use this class.
+`sf::Angle` represents an angle in a unit-agnostic manner so you can reason about angles in terms of degrees or radians.
```

Comment:
The most important part is to explain what `sf::Angle` is, so I'd write it more like this:

```suggestion
[`sf::Angle`](https://www.sfml-dev.org/todo/angle.php "sf::Angle documentation") represents an angle in a unit-agnostic manner so you can reason about angles in terms of degrees or radians.
SFML defines angles by following the [right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule).
This means that if the X-axis points to the right in the window and the Y-axis points down, then a positive angle rotates clockwise.
All SFML classes that use angles work with `sf::Angle`.
```